scripts/fix_async_docstrings.py

```diff
@@ -0,0 +1,51 @@
+import re
+from pathlib import Path
+
+from redbaron import RedBaron  # type: ignore
```

Comment:
Why type: ignore?

Reply:
Because `redbaron` doesn't have typings, and I couldn't find any type stubs (like e.g. `types-setuptools`), and `mypy` was complaining that it can't find any types for the package.

Reply:
makes sense